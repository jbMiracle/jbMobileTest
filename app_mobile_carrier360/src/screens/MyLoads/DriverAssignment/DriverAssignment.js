import React, { Component } from 'react';
import { View } from 'react-native';
import { Text } from 'native-base';

class DriverAssignment extends Component {
  render() {
    return (
      <View testID='driver_assignment' style={ { flex: 1, justifyContent: 'center', alignItems: 'center' } }>
        <Text>Driver Assignment</Text>
      </View>
    );
  }
}

export default DriverAssignment;
