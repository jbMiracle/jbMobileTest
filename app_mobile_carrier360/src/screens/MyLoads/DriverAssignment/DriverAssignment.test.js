import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import DriverAssignment from '../DriverAssignment/DriverAssignment';

it('can render an empty component', () => {
  const renderer = new ShallowRenderer();

  renderer.render(<DriverAssignment />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
