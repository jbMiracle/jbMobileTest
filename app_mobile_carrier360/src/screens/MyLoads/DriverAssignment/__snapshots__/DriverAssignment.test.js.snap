// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render an empty component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="driver_assignment"
>
  <Styled(Text)
    virtual={undefined}
  >
    Driver Assignment
  </Styled(Text)>
</View>
`;
